### DISCUSSION TITLE
Add tooltips to dashboard metrics

### DISCUSSION THREAD

**Discussion by RobertJoonas** [plausible/analytics#1386] [Anything Else]
Currently, when I don't know what a metric such as "Bounce Rate" means, I have to navigate to another page in the docs to find out.

It would be nice if I could read a short tooltip note directly from the dashboard to quickly understand what different metrics mean.

(feedback via email)

**Comment by ukutaht** [comment]
Yeah, tooltips with a one-line explanation with a link to the relevant doc page would be ideal
